sqlmesh/core/model/definition.py

```diff
@@ -1468,4 +1468,7 @@ def _is_udtf(expr: exp.Expression) -> bool:
     "tags": lambda value: (
         exp.to_identifier(value[0]) if len(value) == 1 else exp.Tuple(expressions=value)
     ),
+    "primary_key": lambda value: (
+        exp.to_identifier(value[0]) if len(value) == 1 else exp.Tuple(expressions=value)
```

Comment:
This looks identical to line 1469. Should we have a function for this?

Reply:
I wonder whether doing `exp.Tuple(expressions=value)` is correct in this context. Why are we not parsing each expression in `value` into an identifier as well?

Reply:
It seems all that this is used for is converting meta fields into AST nodes for the purposes of rendering a definition. So having a Union output of Tuple or Identifier based on the input being multiple values or one value seems secondary. At least at a glance, though I am all for less union types, since its simple later on.

Reply:
Unless you mean:
`exp.Tuple(expressions=exp.to_identifier(value))` 

In which case, makes sense actually 🙃

Reply:
i think the ast here is wrong, it should be exp.to_column()